cnf-certification-test/identifiers/remediation.go

```diff
@@ -152,6 +152,8 @@ const (
 
 	ProjectedVolumeServiceAccountRemediation = `Ensure that pods do not use projected volumes and service account tokens`
 
+	LimitedUseOfExecProbesRemediation = `The entire cluster's worth of pods needs to have less than or equal to 10 exec probes.`
```

Comment:
"The entire cluster's worth of pods needs to have less than or equal to 10 exec probes." -> "Reduce the number of exec probes in the cluster for this CNF to less than 10. Increase the update period of the exec probe to be superior or equal to 10 seconds.